src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
It was there, maybe a refresh issue. Anyway, I've cleaned up the history and the stack declaration and usage have been mode to ILToCppImporter.